src/authorizations/utils/permissions.ts

```diff
@@ -185,6 +185,14 @@ export const formatApiPermissions = (permissions, orgID, orgName) => {
             type: key,
           },
         })
+      } else if (key === 'users') {
+        apiPerms.push({
+          action: 'read',
+          resource: {
+            id: meID,
+            type: key,
+          },
+        })
```

Comment:
@wbaker85 this looks right to you, yeah?

Reply:
The "all access" token also has the user `name` field in the permission. Will this one have that? I see from the video that it does so it must be getting added somewhere else?